Scripts/script-emailFieldTriggered.yml

```diff
@@ -0,0 +1,21 @@
+commonfields:
+  id: emailFieldTriggered
+  version: -1
+name: emailFieldTriggered
+script: |-
+  var users = executeCommand('getUsers', {});
+  for (var i = 0; i < users[0].Contents.length; i++) {
+      if (incidents[0].owner === users[0].Contents[i].name) {
+          var email = users[0].Contents[i].email;
+          break;
+      }
+  }
+  var body = 'Hello,\nField ' + args.name + ' was changed from ' + args.old + ' to ' + args.new + ' in incident ' + incidents[0].name + '.\n--DBot';
+  var subject = 'Field ' + args.name + ' was triggered';
+  executeCommand('SendEmail', {'to':email, 'subject':subject, 'body':body});
```

Comment:
check if email exists, then send email. Because incident can be without  owner and then email will be undefined

Reply:
check this scenario incident without owner